frontend/web/components/pages/OrganisationSettingsPage.js

```diff
@@ -26,7 +26,7 @@ import PageTitle from 'components/PageTitle'
 import { getStore } from 'common/store'
 import { getRoles } from 'common/services/useRole'
 
-const widths = [450, 150, 100]
+const widths = [450, 150, 100, 250, 235, 255]
```

Comment:
I'd quite like to understand why we define these widths here rather than just directly in the style objects below. 

It also would make sense to my somewhat picky brain if these were ordered, either smallest to largest or largest to smallest, I'm not _that_ fussy!

Reply:
It seems that initially this array was used to give sizes to the panelSearch component, as I worked on that component I added those values ​​to the array.

Re-ordered.